src/pages/studyView/charts/barChart/CustomBinsModal.tsx

```diff
@@ -57,12 +57,16 @@ export default class CustomBinsModal extends React.Component<
             this.currentBinsValue = newBins.join(`${this.binSeparator} `);
         }
 
+        this.props.store.generateBinsConfig.binSize = this.currentGenerateBinsConfig.binSize;
+        this.props.store.generateBinsConfig.anchorValue = this.currentGenerateBinsConfig.anchorValue;
+
```

Comment:
Better create a new callback updateGenerateBins() and involke it. This way, the state will be changed (an @action) by the handler.

Reply:
Done